var/spack/repos/builtin/packages/py-jsonschema/package.py

```diff
@@ -12,9 +12,15 @@ class PyJsonschema(PythonPackage):
     homepage = "http://github.com/Julian/jsonschema"
     url      = "https://pypi.io/packages/source/j/jsonschema/jsonschema-2.6.0.tar.gz"
 
+    version('3.0.1', sha256='0c0a81564f181de3212efa2d17de1910f8732fa1b71c42266d983cd74304e20d')
     version('2.6.0', sha256='6ff5f3180870836cae40f06fa10419f557208175f13ad7bc26caa77beb1f6e02')
     version('2.5.1', sha256='36673ac378feed3daa5956276a829699056523d7961027911f064b52255ead41')
 
+    depends_on('python@2.6:2.7.999,3.4:')
+    depends_on('python@2.7:2.7.999,3.5:', when='@2.6.0:')
```

Comment:
Where did you find this? Also, they should be `type=('build', 'run')`

Reply:
https://github.com/Julian/jsonschema/commit/2387d08e05bdd9d3a65a01eb9d0c97be8c43a6bd . Will fix the type.